frontend/src/component/changeRequest/ChangeRequest/Changes/Change/EditChange.tsx

```diff
@@ -172,6 +173,14 @@ export const EditChange = ({
                             )}
                             tab={tab}
                             setTab={setTab}
+                            StrategyVariants={
```

Comment:
We pass this in from the outside instead of keeping it inside the form, because create and edit now needs to have different behaviors.